.github/workflows/benchmark.yml

```
@@ -0,0 +1,145 @@
+      - name: Comment on commit
+        if: github.event_name == 'push'
+        continue-on-error: true
+        uses: peter-evans/commit-comment@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          body-path: benchmarks/results.md
+      
+      - name: Comment on pull request
+        if: github.event_name == 'pull_request'
+        continue-on-error: true
+        uses: actions/github-script@v6
```

actionlint (1.7.12)

[error] the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)